packages/devtools_app/lib/src/shared/diagnostics/dart_object_node.dart

```diff
@@ -276,13 +276,9 @@ class DartObjectNode extends TreeNode<DartObjectNode> {
   bool get isExpandable {
     final theRef = ref;
     final instanceRef = theRef?.instanceRef;
-    if (theRef is ObjectReferences) {
-      if (instanceRef?.length == 0) return false;
-      return theRef.refNodeType.isRoot ||
-          children.isNotEmpty ||
-          childCount > 0 ||
-          !isPrimitiveInstanceKind(instanceRef?.kind);
-    }
+
+    if (isRootForReferences(ref)) return true;
+
```

Comment:
there is a logic change here - is it intentional? We used to return early from `isExpandable` for the ObjectReferences case, and now we do not. Now we only return if true, and if false, we continue through the rest of the isExpandable getter. 

Perhaps instead, this should be 
```
if (theRef is ObjectReferences) {
  return isRootForReferences(ref);
}
```

Reply:
Yes, I found bug: we do not expand item, if there is no live reference, while we want to show static references from snapshot in this case.